terraformInstall.sh

```diff
@@ -67,8 +68,15 @@ echo "Updating package lists..."
 sudo apt update
 
 ## CONDITIONAL 6
-echo "Installing Terraform..."
-sudo apt-get install terraform
+
+# Check if Terraform is installed
+if ( command -v terraform > /dev/null 2>&1 )
```

Comment:
Would you want to see the output of this command in case something goes wrong? Anytime I see a redirect to /dev/null it makes me worry we're losing information that could be useful for debugging.